From bac10e3e0c3f715d6effe007e393a9a867eaa71a Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Fri, 25 Jun 2021 22:03:38 -0300
Subject: [PATCH 092/233] [FIO extras] cmd: bootz: don't require HAB verify on
 closed devices

NXP BSP assumes that when HAB is enabled (required on closed devices)
only signed images are allowed to boot, which is not really useful when
debugging or when using lmp-base.

We're safe to drop this requirement as we force fitimage validation on
LmP.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 cmd/bootz.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/cmd/bootz.c b/cmd/bootz.c
index 78c21d47400..4f024bde5fe 100644
--- a/cmd/bootz.c
+++ b/cmd/bootz.c
@@ -57,14 +57,6 @@ static int bootz_start(struct cmd_tbl *cmdtp, int flag, int argc,
 	if (bootm_find_images(flag, argc, argv, images->ep, zi_end - zi_start))
 		return 1;
 
-#ifdef CONFIG_IMX_HAB
-	extern int authenticate_image(
-			uint32_t ddr_start, uint32_t raw_image_size);
-	if (authenticate_image(images->ep, zi_end - zi_start) != 0) {
-		printf("Authenticate zImage Fail, Please check\n");
-		return 1;
-	}
-#endif
 	return 0;
 }
 
-- 
2.40.1

